1601303248587	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1601303248590	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1601303248590	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1601312475404	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1601312475405	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1601312475409	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1601312475410	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1601312478621	Sync.Service	DEBUG	User-Agent: Firefox/81.0 (Linux x86_64) FxSync/1.83.0.20200925100249.desktop
1601312478621	Sync.Service	INFO	Starting sync at 2020-09-28 13:01:18 in browser session s5t_ymtHJChD
1601312478621	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1601312478625	Sync.Status	INFO	Resetting Status.
1601312478625	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1601312478625	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1601312478625	FirefoxAccounts	DEBUG	getOAuthToken enter
1601312478626	FirefoxAccounts	TRACE	getCachedToken returning cached token
1601312478626	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1601312478626	Sync.BrowserIDManager	INFO	Getting keys
1601312478627	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1601312478627	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1601312478627	Sync.BrowserIDManager	DEBUG	Getting a token using OAuth
1601312478627	FirefoxAccounts	DEBUG	getAccessToken enter
1601312478627	FirefoxAccounts	DEBUG	getOAuthToken enter
1601312478628	FirefoxAccounts	TRACE	getCachedToken returning cached token
1601312478628	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1601312478628	FirefoxAccounts	DEBUG	Using cached scopedKeys data for https://identity.mozilla.com/apps/oldsync
1601312478628	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1601312478628	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1601312478630	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1601312478630	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]"}
1601312478631	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1601312478631	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: TypeError: err.response is undefined(resource://services-sync/browserid_identity.js:508:13) JS Stack trace: _fetchTokenUsingOAuth/<@browserid_identity.js:508:13
1601312478632	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1601312478632	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1601312478632	Sync.BrowserIDManager	ERROR	Failed to fetch a token for authentication: TypeError: err.response is undefined(resource://services-sync/browserid_identity.js:508:13) JS Stack trace: _fetchTokenUsingOAuth/<@browserid_identity.js:508:13
1601312478633	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1601312478635	Sync.BulkKeyBundle	INFO	BulkKeyBundle being created for undefined
1601312478636	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/164549947/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1601312478636	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1601312478636	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1601312478636	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1601312478637	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1601312478637	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1601312478642	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
